felix/dataplane/linux/bpf_ep_mgr.go

```diff
@@ -823,6 +877,25 @@ func (m *bpfEndpointManager) updateWEPsInDataplane() {
 	})
 }
 
+func (m *bpfEndpointManager) wepIfaceReady(name string, ready bool) {
```

Comment:
Feels like this method should key off `iface.dpState.isReady` for symmetry with the case in `onInterfaceUpdate`: ideally, whenever we change either flag we should rerun some common code to update the map.  However, `iface.dpState.isReady` never seem to get set back to false, even after an error?  Maybe that's wrong!